test/test_modified_scripts.py

```diff
@@ -37,10 +32,9 @@ def setup_module():
 def get_modified_scripts():
     """Get modified script list, using version.txt in repo and master upstream"""
 
-    os.chdir(retriever_root_dir)
     modified_list = []
-    version_file = requests.get("https://raw.githubusercontent.com/weecology/retriever/master/version.txt")
-    local_repo_scripts = get_script_version()  # local repo versions
+    version_file = requests.get("https://raw.githubusercontent.com/harshitbansal05/retriever/development/version.txt")
```

Comment:
same as above, let's make this the retriever-recipes-dev branch that was just merged in

Reply:
Again, I have updated the url for the recipes repository. I didn't update this one so the pr could be tested prior to merging it in the retriever repo.

Reply:
sure that makes sense, but is there a reason why you didn't use the official weecology retriever-recipes version.txt? (https://raw.githubusercontent.com/weecology/retriever-recipes/retriever-recipes-dev/version.txt)

![image](https://user-images.githubusercontent.com/15080652/59611870-1e809100-90ea-11e9-9e19-225fb42290c8.png)